src/bin/traditional.rs

```diff
@@ -45,6 +49,85 @@ fn hi(_: PotatoRequest) -> PotatoResponse {
         .add_header("Content-Length", &body.len().to_string())
 }
 
+fn simple_add(req: PotatoRequest) -> PotatoResponse {
+    let res = PotatoResponse::new();
+
+    let num_str = req.body.unwrap();
```

Comment:
Don't unwrap body. Handle the case where there is no body provided and return 400 Bad request.